script/lib/transifex.py

```diff
@@ -261,6 +261,28 @@ def get_fingerprint_for_xtb(message_tag):
     # To avoid negative ids we strip the high-order bit
     return str(fp & 0x7fffffffffffffffL)
 
+def is_translateable_string(grd_file_path, message_tag):
+    if message_tag.get('translateable') != 'false':
+        return True
+    # Check for exceptions that aren't translateable in Chromium, but are made
+    # to be translateable in Brave. These can be found in the main function in
+    # brave/script/chromium-rebase-l10n.py
+    head, tail = os.path.split(grd_file_path)
```

Comment:
nit: how about `filename = os.path.basename(grd_file_path)`, I think it's more intuitive but I'm fine with using split too.

Reply:
Fixed.